Makefile

```diff
@@ -117,7 +117,7 @@ lint:
 
 vet:
 	@echo "vet"
-	@retool do govet -all -shadow $$($(PACKAGE_DIRECTORIES)) 2>&1 | $(FAIL_ON_STDOUT)
+	@go vet -all -shadow $(PACKAGES) 2>&1 | $(FAIL_ON_STDOUT)
```

Comment:
https://github.com/pingcap/tidb/pull/7197
It's changed in this pr. Should we change it back?

Reply:
IMHO, the introduce of `retool` in https://github.com/pingcap/tidb/pull/7197 is good for project management, it makes TiDB survive from third-party tool changes, so we'd better keep it. @winoros 

I don't know why `govet` fail here under the latest Go1.11, but the standard go vet tool works, this change is a workaround.